openfpga/src/tcl_api/openfpga_wrap.cpp

```diff
@@ -0,0 +1,2372 @@
+/* ----------------------------------------------------------------------------
```

Comment:
This file should not be checked in. It should be generated on the fly by cmake. See above.

Reply:
I will add necessary commands in `cmake` to generate the wrapper.cpp